RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,65 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation, and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, where we’d like to grow in future. 
+
+
+## Core Strategy Statement:  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## What we do
+
+First and foremost, we create templates with supporting meta-documents that underpin the templates.
+
+As part of this work, we engage in many other initiatives around docs advocacy, docs education, and docs tooling. 
+
+We produce content to address the goals of engaging in our community and helping run the project; and learning and gaining experience in the realm of tech writing.
```

Comment:
Does "helping run the project" refer to: 
-  content that is intended to run TGDP or 
- content that is intended to run the open-source projects of the people that come to us for advice?

Reply:
I intended this to mean "project docs" (internal docs/handbook). 
I like to think that our project docs could be open for other projects to copy and use. But less formal than our templates (like without the meta-docs).

Do you think I need to add/change wording here to make the meaning clearer?

Reply:
I think this got resolved in a comment from Alyssa.